GoogleDataTransport/GDTCORLibrary/Public/GDTCORPlatform.h

```diff
@@ -69,20 +69,52 @@ typedef NS_ENUM(NSInteger, GDTCORNetworkMobileSubtype) {
   GDTCORNetworkMobileSubtypeLTE = 11,
 };
 
+/** The different possible reachabilityFlags option. */
+typedef NS_ENUM(NSInteger, GDTCORReachabilityFlagsOption) {
```

Comment:
You need to add a unit test that check that these values are the same as iOS's

Reply:
Remove flags definition on non-watchos